Examples/Python/python/acts/examples/itk.py

```diff
@@ -390,32 +393,35 @@ def itkSeedingAlgConfig(
         interactionPointCut = True
         impactMax = 2 * u.mm
         zBinsCustomLooping = [
-            1,
             2,
             3,
             4,
+            5,
+            12,
             11,
             10,
             9,
-            8,
-            6,
-            5,
             7,
+            6,
+            8,
         ]  # enable custom z looping when searching for SPs, must contain numbers from 1 to the total number of bin in zBinEdges
         zBinNeighborsTop = [
             [0, 0],
             [-1, 0],
+            [-2, 0],
```

Comment:
why [-2, 0] and not [-1, 0] ?

Reply:
What this PR does is taking the first bin #1 and diverse it in two bins #1 and #2. 

All the other bins then also increase by 1.

Now, in order to keep the same results as before, we need the former bin #2 (which now is #3) to search the top candidate in the two bins on the left. Thus the -2.

The same reasoning applies to the rightmost side of the detector

Reply:
Thanks! I see